File: Java/02- Relacionando Classes/Profissões/Motoboy.java

package com.mycompany.mavenproject4;


public class Motoboy extends Funcionario{
    private String cnh;

    public Motoboy(String cnh, String nome, String cpf, String rg, double salario) {
        super(nome, cpf, rg, salario);
        this.cnh = cnh;
    }

    public String getCnh() {
        return cnh;
    }

    public void setCnh(String cnh) {
        this.cnh = cnh;
    }

    @Override
    public String toString() {
        return "\nDados do Motoboy: " + 
                super.toString() +
               "\nCNH: " + cnh;
    }
    
    
}

File: Python/04- POO/03- Atividade Pontuada/models/enums/trabalho/motoboy.py
from models.funcionario import Funcionario

class Motoboy(Funcionario):
    def __init__(self, nome, telefone, email, endereco, cpf, rg, data_nascimento, sexo, matricula, setor, salario, cnh: str) -> None:
        super().__init__(nome, telefone, email, endereco, cpf, rg, data_nascimento, sexo, matricula, setor, salario)
        self.cnh = cnh

    def salario_final(self):
        return self.salario

    def __str__(self) -> str:
        return (f"{super().__str__()}" 
                f"\nCNH: {self.cnh}" 
               )